README.md

```diff
@@ -33,7 +33,7 @@ The Jellyfin package is in the AUR at https://aur.archlinux.org/packages/jellyfi
 
 A package repository is available at https://repo.jellyfin.org. To use it:
 
-0. Install the `dotnet-runtime-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+0. Install the `dotnet-runtime-2.2` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.2).
 0. Import the GPG signing key (signed by Joshua):
```

Comment:
Is there any reason to link the key to a specific person, or should there just be 1 Jellyfin signing key?

Reply:
Open an issue for that.

Reply:
I set the repo up as quick as I could, so signing it as myself was the path of lease resistance. Something to revisit once we stabilize a bit more.

Reply:
Let's tag this in separate issue for tracking - #150